packages/flutter_tools/lib/executable.dart

```diff
@@ -95,7 +95,10 @@ Future<void> main(List<String> args) async {
     DevicesCommand(),
     DoctorCommand(verbose: verbose),
     DowngradeCommand(),
-    DriveCommand(verboseHelp: verboseHelp),
+    DriveCommand(verboseHelp: verboseHelp,
+      fileSystem: globals.fs,
+      logger: globals.logger,
```

Comment:
No, the only interfaces that can't be injected at this point are those that depend on Cache/Artifacts/Version since those are still constructed when the first FlutterCommand is executed